tests/serialization/test_dag_serialization.py

```diff
@@ -915,68 +944,6 @@ def execute(self, context: Context):
         )
         assert expected_err_msg in caplog.text
 
-    def test_extra_serialized_field_and_multiple_operator_links(self, dag_maker):
```

Comment:
This test was broadly a duplicate of test_extra_serialized_field_and_operator_links -- so I made that parameterized instead.